### DISCUSSION TITLE
can't fit any time series model (all skipped)

### DISCUSSION THREAD

**Discussion by michelidiego** [autogluon/autogluon#4182] [Q&A]
Hi Everyone, 

I'm training that I've used before only the initial conditions have changed. Initially I run the model using 2 weeks of training data to forecast one day. This works just fine.

I now wanted to use a similar model with the same initial data but only this time I'm using 6 months of history (training data) and trying to forecast 21 days. 

I'm using ffill method to fill the training data's missing values.

This is the error I'm getting:
```

AutoGluon will gauge predictive performance using evaluation metric: 'RMSE'
  This metric's sign has been flipped to adhere to being higher_is_better. The metric score can be multiplied by -1 to get the metric value.
===================================================

Starting training. Start time is 2024-05-07 19:28:43
Models that will be trained: ['SeasonalNaive', 'RecursiveTabular', 'DirectTabular', 'CrostonSBA', 'NPTS', 'DynamicOptimizedTheta', 'AutoETS', 'AutoARIMA', 'Chronos[base]', 'TemporalFusionTransformer', 'DeepAR', 'PatchTST']
Training timeseries model SeasonalNaive. 
  Warning: SeasonalNaive/W0 failed for 1 time series (1.3%). Fallback model SeasonalNaive was used for these time series.
  Warning: Exception caused SeasonalNaive to fail during training... Skipping this model.
  Predictions contain NaN values.
Training timeseries model RecursiveTabular. 
  Warning: Exception caused RecursiveTabular to fail during training... Skipping this model.
  Label column cannot contain non-finite values (NaN, Inf, Ninf). First invalid label at train_data idx: 3214
Training timeseries model DirectTabular. 
  Warning: Exception caused DirectTabular to fail during training... Skipping this model.
  Label column cannot contain non-finite values (NaN, Inf, Ninf). First invalid label at train_data idx: 3214
Training timeseries model CrostonSBA. 
  Warning: CrostonSBA/W0 failed for 1 time series (1.3%). Fallback model SeasonalNaive was used for these time series.
  Warning: Exception caused CrostonSBA to fail during training... Skipping this model.
  Predictions contain NaN values.
Training timeseries model NPTS. 
  Warning: NPTS/W0 failed for 1 time series (1.3%). Fallback model SeasonalNaive was used for these time series.
  Warning: Exception caused NPTS to fail during training... Skipping this model.
  Predictions contain NaN values.
Training timeseries model DynamicOptimizedTheta. 
  Warning: DynamicOptimizedTheta/W0 failed for 1 time series (1.3%). Fallback model SeasonalNaive was used for these time series.
  Warning: Exception caused DynamicOptimizedTheta to fail during training... Skipping this model.
  Predictions contain NaN values.
Training timeseries model AutoETS. 
  Warning: AutoETS/W0 failed for 1 time series (1.3%). Fallback model SeasonalNaive was used for these time series.
  Warning: Exception caused AutoETS to fail during training... Skipping this model.
  Predictions contain NaN values.
Training timeseries model AutoARIMA. 
  Warning: AutoARIMA/W0 failed for 1 time series (1.3%). Fallback model SeasonalNaive was used for these time series.
  Warning: Exception caused AutoARIMA to fail during training... Skipping this model.
  Predictions contain NaN values.
Training timeseries model Chronos[base]. 
  Warning: Exception caused Chronos[base] to fail during training... Skipping this model.
  Chronos[base]/W0 requires a GPU to run, but no GPU was detected. Please make sure that you are using a computer with a CUDA-compatible GPU and `import torch; torch.cuda.is_available()` returns `True`.
Training timeseries model TemporalFusionTransformer. 
  Warning: Exception caused TemporalFusionTransformer to fail during training... Skipping this model.
  /home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/torchaudio/lib/libtorchaudio.so: undefined symbol: _ZN3c104impl3cow11cow_deleterEPv
Training timeseries model DeepAR. 
  Warning: Exception caused DeepAR to fail during training... Skipping this model.
  /home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/torchaudio/lib/libtorchaudio.so: undefined symbol: _ZN3c104impl3cow11cow_deleterEPv
Training timeseries model PatchTST. 
  Warning: Exception caused PatchTST to fail during training... Skipping this model.
  /home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/torchaudio/lib/libtorchaudio.so: undefined symbol: _ZN3c104impl3cow11cow_deleterEPv
Not fitting ensemble as no models were successfully trained.
Training complete. Models trained: []
Total runtime: 80.39 s
Trainer has no fit models that can predict.

```
Any thoughts on this?

I also tried to change the training data's history and the only thing that changed is that, in almost every model, I got this error:

`Label column cannot contain non-finite values (NaN, Inf, Ninf)`

Thank you for your help

**Comment by shchur** [comment]
Hi @michelidiego, thank you for reporting the problem. Can you please provide a small example for reproducing this issue (code, full log and, ideally, a part of the dataset)?
